test/testutils/syncenginetestutils.cpp

```diff
@@ -254,9 +255,38 @@ bool FileInfo::operator==(const FileInfo &other) const
     // Consider files to be equal between local<->remote as a user would.
     return name == other.name
         && isDir == other.isDir
-        && size == other.size
+        && contentSize == other.contentSize
         && contentChar == other.contentChar
-        && children == other.children;
+            && children == other.children;
+}
+
+bool FileInfo::equals(const FileInfo &other, ComparissonOption opt) const
+{
+    switch (opt) {
+    case ContentIsKing:
+        return *this == other;
+    case IgnoreContentOfDehydratedFiles:
+        if (!isDehydratedPlaceholder && !other.isDehydratedPlaceholder) {
```

Comment:
please also check modtime and other available meta data.

Reply:
which ones, other than `lastModified`?

Reply:
Well you are right we only have the modtime.